core/src/main/kotlin/org/evomaster/core/problem/rest/service/RestFitness.kt

```diff
@@ -74,6 +74,12 @@ open class RestFitness : AbstractRestFitness<RestIndividual>() {
 
             if (a is RestCallAction) {
                 ok = handleRestCall(a, actionResults, chainState, cookies, tokens)
+                /*
+                    the action might be stopped due to e.g., timeout (see [handleRestCall]),
+                    but the property of [stopping] is not handle.
+                    we can also handle the property inside [handleRestCall]
+                 */
+                actionResults.filterIsInstance<RestCallResult>()[i].stopping = !ok
```

Comment:
no, this is not what `stopping` was used for.
see `AbstractRestFitness.handleSaveLocation`

a failed call (eg due to timeout) does NOT prevent making the following calls in the test

Reply:
yes. I found that it is the only place where to handle the property `stopping`. but in the loop of executing rest action, if the `ok` is false, the following actions are not executed (see lines 75 and 88 in `RestFitness`).  do we add another property to control if the following actions from the current were executed?

Reply:
hi. hmmm... you might be right. but i need to think a bit more about this